nginx_vts_exporter.go

```diff
@@ -340,16 +350,17 @@ func fetchHTTP(uri string, timeout time.Duration) func() (io.ReadCloser, error)
 }
 
 var (
-	listenAddress   = flag.String("telemetry.address", ":9113", "Address on which to expose metrics.")
+	listenAddress   = flag.String("telemetry.address", ":9913", "Address on which to expose metrics.")
```

Comment:
Yes, this was a random port number, and it's configurable. But it's better for us to keep the former default port number 9113, isn't it?